### ISSUE TITLE
Gravitychanger mod ID change from gravitychanger too gravity_api

### ISSUE THREAD

**Issue by Stick404** [FallingColors/HexMod#650]
### Modloader

Fabric

### Minecraft version

1.19.2

### Hex Casting version

0.10.3

### Modloader version

Fabric 0.14.21, and a API version of 0.76.0+1.19.2

### Modpack info

The Hexcasting base mod/HexDummy

### The latest.log file

no logs available for this issue

### Issue description

In the fabirc.mod.json (and presumably in the forge version), the mod tries to get the fusionflux gravity API, however that has moved into quilt only; however Qouteall has made their own fork of gravity API (found at Modrinth here https://modrinth.com/mod/gravity-api-fork and GitHub here https://modrinth.com/mod/gravity-api-fork). The only thing that wound need to change is the fabric.mod.json's (and maybe the forge one too), ``"gravitychanger": "0.7.21+fabric"`` too ``"gravity_api": "0.9.0"``. I do not believe that anything else would have to change.

### Steps to reproduce

no steps need to reporduce

**Comment by SamsTheNerd** [comment]
i believe there's a non-quilt only one for 1.19.2 that just had a slightly different modid? I don't think a new 1.19.2 update is coming anytime soon though.

For 1.20 we were just planning on moving the interop into hexbound i think if June was still ok with that? 

I'm not opposed to switching to the fork for 1.20 though, especially if the APIs are the same. Ideally we'd be able to support both the fork and the original, but I guess that depends on if their APIs are the same or not

**Comment by Stick404** [comment]
> i believe there's a non-quilt only one for 1.19.2 that just had a slightly different modid? I don't think a new 1.19.2 update is coming anytime soon though.

yup, those are the ones I linked too, sadly its ether 1.19.0 or 1.19.4 so IDK if one of those works
